--- aux/zeekctl/cmake/InstallSymlink.cmake-orig
+++ aux/zeekctl/cmake/InstallSymlink.cmake
@@ -17,25 +17,17 @@ macro(InstallSymlink _filepath _sympath)
     get_filename_component(_symname ${_sympath} NAME)
     get_filename_component(_installdir ${_sympath} PATH)
 
-    if (BINARY_PACKAGING_MODE)
-        execute_process(COMMAND "${CMAKE_COMMAND}" -E create_symlink
-                        ${_filepath}
-                        ${CMAKE_CURRENT_BINARY_DIR}/${_symname})
-        install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${_symname}
-                DESTINATION ${_installdir})
-    else ()
-        # scripting the symlink installation at install time should work
-        # for CMake 2.6.x and 2.8.x
-        install(CODE "
-            if (\"\$ENV{DESTDIR}\" STREQUAL \"\")
-                execute_process(COMMAND \"${CMAKE_COMMAND}\" -E create_symlink
-                                ${_filepath}
-                                ${_installdir}/${_symname})
-            else ()
-                execute_process(COMMAND \"${CMAKE_COMMAND}\" -E create_symlink
-                                ${_filepath}
-                                \$ENV{DESTDIR}/${_installdir}/${_symname})
-            endif ()
-        ")
-    endif ()
+    # scripting the symlink installation at install time should work
+    # for CMake 2.6.x and 2.8.x
+    install(CODE "
+        if (\"\$ENV{DESTDIR}\" STREQUAL \"\")
+            execute_process(COMMAND \"${CMAKE_COMMAND}\" -E create_symlink
+                            ${_filepath}
+                            ${_installdir}/${_symname})
+        else ()
+            execute_process(COMMAND \"${CMAKE_COMMAND}\" -E create_symlink
+                            ${_filepath}
+                            \$ENV{DESTDIR}/${_installdir}/${_symname})
+        endif ()
+    ")
 endmacro(InstallSymlink)
